Update URL to WDK10.0.17134

OwenSmith opened the pull request:
Updates the download URL and checksum to WDK 10.1.17134.1

Signed-off-by: Owen Smith <owen.smith@citrix.com>

OwenSmith committed:
Update URL to WDK10.1.17134.1

Updates the download URL and checksum to WDK 10.1.17134.1

Signed-off-by: Owen Smith <owen.smith@citrix.com>

nioncode commented on windowsdriverkit10.nuspec:
```diff
@@ -4,10 +4,10 @@
     <id>windowsdriverkit10</id>
     <title>Windows Driver Kit 10</title>
-    <version>10.0.10586</version>
+    <version>10.1.17134.1</version>
```
Shouldn't this be 10.**0**.17134.1?

OwenSmith replied:
It seems even Microsoft are confused about this.
wdksetup.exe is versioned 10.1.17134.1, which installs WDK 10.0.17134.1 (according to Add/Remove programs) , which also displays version 10.1.17134.1 as its package version...
![wdk-version](https://user-images.githubusercontent.com/4749846/44101405-45cea38e-9fdf-11e8-88bf-e3f399d4021c.png)

nioncode replied:
How does it show up in Visual Studio? I think this should be the definitive version.

OwenSmith replied:
Visual Studio 2017 reports the plugin version 10.0.17110.0
The WDK includes are under a versioned path, using 10.0.17134.0

I'd be inclinded to suggest the version should be 10.0.17134 with no 4th digit if thats an acceptable nuspec entry

nioncode replied:
Sounds good. Actually I now see I had the same issue when I first released the package and switched from .1. to .0.

OwenSmith committed:
Fix version to 10.0.17134

Theres a bit of confusion as to the exact version the WDK should use
wdksetup version is 10.1.17134.1
WDK installer uses 10.0.17134.1
headers installed to versioned location 10.0.17134.0
Visual Studio 2017 plugin reports version 10.0.17110.0

Signed-off-by: Owen Smith <owen.smith@citrix.com>

OwenSmith commented:
Updated PR title to reflect updated versioning

OwenSmith commented:
Any update on this PR?

nioncode commented:
Thanks, I merged it just now and will publish it on the chocolatey repo soon.
